test/TargetFunctions.sol

```diff
@@ -255,6 +276,34 @@ abstract contract TargetFunctions is Properties {
         // --- Ghost data after ---
         last_action = LastAction.DONATE;
         __totalAssetAfter = woeth.totalAssets();
+        __oeth_balanace_of_woeth = oeth.balanceOf(address(woeth));
+        __hardAssets = woeth.hardAssets();
+        __yieldAssets = woeth.yieldAssets();
+        __yieldEnd = woeth.yieldEnd();
+    }
+
+    /// @notice Handle pass time on chain
+    /// @param _amount Amount of time to pass. 1 Day is maximum, since that is also the 
+    ///        maximum yield time 
+    function handler_pass_time(uint88 _amount) public {
```

Comment:
As is `MAX_YIELD_TIME` will be `86400` maximum, you can reduce the uint88 to uint24. 
This will help the fuzzer to generate better input.

Reply:
And maybe we can replace `_amount` by `_duration` or something like this?

Reply:
good thinking thanks! https://github.com/OriginProtocol/woeth-fuzzing-tests/pull/1/commits/6e066b44ea2ce3cc35e7bef7499820382db058d9

Reply:
Interestingly forge fuzzer doesn't work with `uint24`. Checked the docs and couldn't find the list of supported types.